module/VuFind/src/VuFind/View/Helper/Notifications/Notifications.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+/**
+ * Notifications view helper
+ *
+ * PHP version 8
+ *
+ * Copyright (C) effective WEBWORK GmbH 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Johannes Schultze <schultze@effective-webwork.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\View\Helper\Notifications;
```

Comment:
We generally organize our view helpers on the basis of theme. Unless you're planning to have a group of notifications-related helpers, I'd suggest moving this to the VuFind\View\Helper\Root namespace.